### PR TITLE
[DOCS] Plugin Installation for Windows

### PR DESCRIPTION
**Pull request by pickypg**
This shows an example of how to install a plugin on Windows, which is not as obvious at I would have expected.

### CODE REVIEW

File: docs/plugins/plugin-script.asciidoc [hunk 1]
```diff
@@ -64,13 +64,35 @@ sudo bin/elasticsearch-plugin install [url] <1>
 -----------------------------------
 <1> must be a valid URL, the plugin name is determined from its descriptor.
 
-For instance, to install a plugin from your local file system, you could run:
+==== Unix File System
```
Comment (nik9000) [inline docs/plugins/plugin-script.asciidoc]: I think having a file that has three levels of titles in it upsets asciidoc. Check before merging please.

File: docs/plugins/plugin-script.asciidoc [hunk 2]
```diff
@@ -64,13 +64,35 @@ sudo bin/elasticsearch-plugin install [url] <1>
...
+To install a plugin from your local file system at `/path/to/plugin.zip`, you could run:
 
 [source,shell]
 -----------------------------------
 sudo bin/elasticsearch-plugin install file:///path/to/plugin.zip
 -----------------------------------
 
+==== Windows File System
+
+On Windows, the `file` path is much easier to work with using absolute paths only. Absolute paths
```
Comment (nik9000) [inline docs/plugins/plugin-script.asciidoc]: Looks like we recommend using an absolute path for unix file systems as well. Maybe drop the mention of absolute paths entirely?

File: docs/plugins/plugin-script.asciidoc [hunk 3]
```diff
@@ -64,13 +64,35 @@ sudo bin/elasticsearch-plugin install [url] <1>
...
+==== Unix File System
+
+To install a plugin from your local file system at `/path/to/plugin.zip`, you could run:
 
 [source,shell]
 -----------------------------------
 sudo bin/elasticsearch-plugin install file:///path/to/plugin.zip
 -----------------------------------
 
+==== Windows File System
```
Comment (clintongormley) [inline docs/plugins/plugin-script.asciidoc]: I'm not sure that the explanatory para is really necessary.  Couldn't it just be replaced with:

    Or on Windows:

    ...example...

Comment (pickypg) [inline docs/plugins/plugin-script.asciidoc]: I wonder if I should switch it to break it down like

```asciidoc
Unix::
+
To install a plugin from your local file system at `/path/to/plugin.zip`, you could run:
+
[source,shell]
-----------------------------------
sudo bin/elasticsearch-plugin install file:///path/to/plugin.zip
-----------------------------------

Windows::
+
To install a plugin from your local file system at `C:\path\to\plugin.zip`, you could run:
+
[source,shell]
-----------------------------------
bin\elasticsearch-plugin install file:///C:\path\to\plugin.zip
-----------------------------------
+
NOTE: Any path that contains spaces must be wrapped in quotes! 

http::
+
[source,shell]
-----------------------------------
sudo bin/elasticsearch-plugin install http://some.domain/path/to/plugin.zip
-----------------------------------

**Comment by karmi** [comment]
Hi @pickypg, we have found your signature in our records, but it seems like you have signed with a different e-mail than the one used in yout Git [commit](https://github.com/elastic/elasticsearch/pull/21671.patch). Can you please add both of these e-mails into your Github profile (they can be hidden), so we can match your e-mails to your Github profile?

**Comment by pickypg** [comment]
Updated @clintongormley. Let me know what you think.

**Comment by rjernst** [comment]
@clintongormley Are you ok with this to be merged?

**Comment by clintongormley** [comment]
I don't think this is correct.  According to https://blogs.msdn.microsoft.com/ie/2006/12/06/file-uris-in-windows/ the file URL should be `file:///C:/foo/bar/baz`

**Comment by dakrone** [comment]
@pickypg can you update this as per Clint's feedback?

**Comment by pickypg** [comment]
FWIW, I tried it on Windows and both work, but it definitely makes sense to go with what Microsoft says is right for their OS.

**Comment by rjernst** [comment]
@clintongormley Ping to review now that this has been updated.

**Review by clintongormley — APPROVED** [review]
LGTM

**Comment by pickypg** [comment]
6.x/6.1: e1e441b
6.0: 4a88a91
5.6: 9ea3421
5.5: eeefbbe
5.4: 08d00fa
5.3: b16899e
5.2: 31cd9fe
5.1: 2e5e881
5.0: 90ed3ca

**Pull request merged**